app/scripts/metamask-controller.js

```diff
@@ -3920,8 +3906,6 @@ export default class MetamaskController extends EventEmitter {
     ///: END:ONLY_INCLUDE_IN
 
```

Comment:
I was able to fix the subscriptions by forwarding logs from the provider to this provider engine, like this:

```suggestion

    // forward notifications from network provider
    provider.on('data', (error, message) => {
      if (error) {
        // This should never happen, this error parameter is never set
        throw error;
      }
      engine.emit('notification', message);
    });
```

Reply:
Moved this up a bit but added here: [469bbed](https://github.com/MetaMask/metamask-extension/pull/16693/commits/469bbed580be40d466cc76a97d7910f727eea5cb)